libs.jaxb/external/jaxb-2.2.5-2-jaxb1-license.txt

```diff
@@ -0,0 +1,445 @@
+Name: JAXB
+Description: JAXB Reference Implementation
+Version: 2.2.5-2
+Origin: Oracle
+URL: https://jaxb.java.net/
+License: CDDL-1.1-BSD-MIT-jaxb1-xjc
+Files: jaxb1-impl-2.2.5-2.jar
+
+=========== Parts of the work are: ===========
```

Comment:
This is odd. Which parts is under which license? I don't believe we have another dependency under a mix of licenses (or do we?).

I assume it's a fat JAR which bundles multiple package names each with different licenses?